packages/vite-plugin-svelte/src/utils/watch.ts

```diff
@@ -54,7 +54,7 @@ export function setupWatchers(
 			});
 		} else {
 			log.info(`svelte config changed: restarting vite server. - file: ${filename}`);
-			server.restart(!!options.experimental?.prebundleSvelteLibraries);
+			server.restart();
```

Comment:
This was removed as `buildStart` will now conditionally determine if prebundling (or force optimize) is needed. Instead of always prebundle on restart.